spec/features/auth_spec.rb

```diff
@@ -149,6 +149,22 @@
     end
   end
 
+  context 'With administrator privileges' do
+    scenario 'user with admin privileges can open list with users' do
+      user = create(:user, email: 'jan.kowalski@gmail.com', password: 'jkpassword', admin: true)
+      sign_in user
+
+      create(:user, email: 'test1@gmail.com', password: 'jkpassword')
```

Comment:
Password is not relevant to the current spec - you can skip it, and use the default one defined in the factory